core/lib/src/rocket.rs

```diff
@@ -66,40 +67,38 @@ pub(crate) struct Token;
 impl Rocket {
     #[inline]
     fn _mount(&mut self, base: Origin<'static>, routes: Vec<Route>) {
-        info!("{}{} {}{}",
-              Paint::emoji("🛰  "),
-              Paint::magenta("Mounting"),
-              Paint::blue(&base),
-              Paint::magenta(":"));
+        let span = tracing::info_span!("mounting", at = %Paint::blue(&base), "{} Mounting", Paint::emoji("🛰 "),);
+        let _e = span.enter();
 
         for route in routes {
             let old_route = route.clone();
             let route = route.map_base(|old| format!("{}{}", base, old))
                 .unwrap_or_else(|e| {
-                    error_!("Route `{}` has a malformed URI.", old_route);
-                    error_!("{}", e);
+                    let span = tracing::error_span!("malformed_uri", "Route `{}` has a malformed URI.", old_route);
+                    error!(parent: &span, "{}", e);
```

Comment:
This doesn't look quite right. Don't you need to enter this span too?

Reply:
using `parent: &span` in the event we record on the next line explicitly sets `span` as the event's parent, which is equivalent to
```rust 
let span = tracing::error_span!(...);
let _e = span.enter();
error!(...);
```

i used `parent` here, rather than entering the span, because the span is only used as a parent for that specific event, and it didn't involve adding another line of code, so i thought it was better to have a slightly smaller diff. it would be fine to enter the span, instead — perhaps i should have done that to make this clearer.

Reply:
i find the other style more explicit
since it doesn't rely on side effect

Reply:
There's no difference in the time stamps that are emitted between an explicit `span.enter()` vs `parent: &span`?